Assets/Scripts/Controllers/EnemyController.cs

```diff
@@ -35,25 +36,30 @@ private void Start()
             //Get bounds of spawnable area
             boundX = spawnableArea.GetComponent<SpriteRenderer>().size.x / 2;
             boundY = spawnableArea.GetComponent<SpriteRenderer>().size.y / 2;
-
-            //Start round - Arbitrary number of enemies to spawn
-            SpawnRound(10);
         }
 
         public void Update()
         {
-            if (livingEnemies.Count == 0)
-            {
-                roundController.LastEnemyDestroyed();
-            }
+            if (!_active) return;
+            if (livingEnemies.Count > 0) return;
+
+            Debug.Log("No enemies remaining");
+            _active = false;
+            roundController.LastEnemyDestroyed();
         }
 
-        public void ClearLivingEnemies()
+        private void ClearLivingEnemies()
```

Comment:
This func is pretty much broken due to iteration issues. SInce it would only be used for debugging, I didn't spend more time here